packages/cactus-core-api/src/main/json/openapi.json

```diff
@@ -1,751 +1,793 @@
 {
-    "openapi": "3.0.3",
-    "info": {
-        "title": "Hyperledger Core API",
-        "description": "Contains/describes the core API types for Cactus. Does not describe actual endpoints on its own as this is left to the implementing plugins who can import and re-use commonly needed type definitions from this specification. One example of said commonly used type definitions would be the types related to consortium management, cactus nodes, ledgers, etc..",
-        "version": "0.2.0",
-        "license": {
-            "name": "Apache 2.0",
-            "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+  "openapi": "3.0.3",
```

Comment:
@Leeyoungone Is it possible that you changed the indentation of the file? I'm seeing almost all lines as changed so I'm thinking it must be some white space related change. If yes, please change the indentation back so that it's easier to review and then in another PR later we can make a dedicated change that does not change anything but the indentation (so that it's also easy to review and does not mask other changes)

Reply:
I think I might have used the "format with" and changed the indentations of my files, I'll make sure to change them back. Thank you very much!!